astropy/utils/tests/test_data.py

```diff
@@ -1092,6 +1090,8 @@ def osraiser(dirnm, linkto, pkgname=None):
                 break
     assert len(partial_warn_msgs) == 0, f'Got some unexpected warnings: {partial_warn_msgs}'
 
+    assert n_warns in (2, 4), f'Expected 2 or 4 warnings, got {n_warns}'
```

Comment:
I am not ready for this rabbit hole this week. As long as test pass, I am ok with this lol